src/components/TodoList/TodoList.js

```diff
@@ -9,16 +10,25 @@ import {
 
 function TodoList() {
   let content = "Lorem ipsum dolor sit amet";
+  const [todoInput, setTodoInput] = useState("");
+  const [list, setList] = useState([]);
   return (
     <StyledTodoList>
       <StyledList>
         <HeadContent>Check List</HeadContent>
         <TodoContainer>
-          <TodoItem checkStatus="checked" content={content}></TodoItem>
-          <TodoItem checkStatus="checked" content={content}></TodoItem>
-          <TodoItem background={"yellow"} content={content}></TodoItem>
+          {/* checkStatus: checked, unchecked, crossed. */}
+          {/* background, content*/}
+          {list.map((item) => {
+            return <TodoItem content={item}></TodoItem>;
```

Comment:
render 1 danh sách luôn nên có key để react render được tối ưu Đức

Reply:
https://www.geeksforgeeks.org/reactjs-keys/#:~:text=A%20%E2%80%9Ckey%E2%80%9D%20is%20a%20special,the%20elements%20in%20the%20lists.